ndk/src/bitmap.rs

```diff
@@ -136,9 +136,13 @@ impl AndroidBitmapInfo {
         self.inner.stride
     }
 
-    pub fn format(&self) -> BitmapFormat {
+    pub fn try_format(&self) -> Option<BitmapFormat> {
         let format = self.inner.format as u32;
-        format.try_into().unwrap()
+        format.try_into().ok()
+    }
+
+    pub fn format(&self) -> BitmapFormat {
+        self.try_format().unwrap()
```

Comment:
The downside is that this now panics on `None` rather than the descriptive error from `TryFromPrimitive` (which hopefully includes the `u32` number that failed to parse).

Reply:
Fair enough. Updated to return the `Result`.